liba/src/external/softfloat/src/f64_to_i64_r_minMag.c

```diff
@@ -90,7 +90,7 @@ int_fast64_t f64_to_i64_r_minMag( float64_t a, bool exact )
         *--------------------------------------------------------------------*/
         sig |= UINT64_C( 0x0010000000000000 );
         absZ = sig>>shiftDist;
-        if ( exact && (absZ<<shiftDist != sig) ) {
+        if ( exact && (absZ<<shiftDist != (int_fast64_t)sig) ) {

```

Comment:
If that's a bug, it should be reported upstream (softfloat). In general we avoid patching upstream code, who usually "knows better".